bounties/php/CMS/1/README.md

```diff
@@ -0,0 +1,17 @@
+## Discription
+encountered **RateLimiting** in login page of LavaLite/CMS Repo while performing little recon to find the cms xss bug that is among the bugs to be fixed in the huntr platform. Lavavite/cms is an open source of Content Management System developed with Laravel framework.
+## POC
```

Comment:
Spelling mistake and please make this a single `#`

Cheers! 🍰

Reply:
done